packages/hooks/src/useIsPasskeySupported.ts

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { platformAuthenticatorIsAvailable } from '@simplewebauthn/browser';
+
+const useIsPasskeySupported = () => {
+    const [is_passkey_supported, setIsPasskeySupported] = React.useState(false);
+    const [is_loading, setIsLoading] = React.useState(true);
+
+    React.useEffect(() => {
+        const checkPasskeySupport = async () => {
```

Comment:
Why did we declare the `checkPasskeySupport` function inside the useEffect?

Reply:
dont know, why not? :)
this function works only once.
any suggestions?